src/core/variables.jl

```diff
@@ -242,6 +242,111 @@ Docs abbreviation: ``y``
 """
 struct HVDCActivePowerReceivedToVariable <: VariableType end
 
+"""
+Struct to dispatch the creation of HVDC Received Reactive Flow From Bus Variables
+
+Docs abbreviation: ``x^r``
+"""
+struct HVDCReactivePowerReceivedFromVariable <: VariableType end
```

Comment:
I don't think we need HVDC specific variables for these reactive power variables on the AC side. We can consolidate with the AC power ones.

Reply:
Agree here, but do we use a meta for ReactivePowerVariable, `from` & `to`?

Reply:
we can use the same model as AC Reactive Power and the key will have the DeviceType